src/standard/configure.html

```diff
@@ -113,18 +115,23 @@
       }
     },
 
-    _configureProperties: function(properties, config) {
-      for (var i in properties) {
-        var c = properties[i];
+    _configureInstanceProperties: function(config) {
```

Comment:
Let's consider having this call `_configValue` to get the `readOnly` check. That will also mean this method should be called before `_aboveConfig` is made.